bpf/bpf_host.c

```diff
@@ -452,8 +452,8 @@ handle_ipv4(struct __ctx_buff *ctx, __u32 secctx,
 	    __u32 ipcache_srcid __maybe_unused, const bool from_host)
 {
 	struct remote_endpoint_info *info = NULL;
+	struct trace_ctx __maybe_unused trace;
```

Comment:
I think these need to be zero-initalized? (Not sure how that fits into `__maybe_unused`..)
```suggestion
	struct trace_ctx __maybe_unused trace = {};
```